examples/jsm/loaders/LDrawLoader.js

```diff
@@ -45,6 +45,9 @@ const COLOR_SPACE_LDRAW = SRGBColorSpace;
 const _tempVec0 = new Vector3();
 const _tempVec1 = new Vector3();
 
+const _edgeMaterialCache = new WeakMap();
+const _conditionalEdgeMaterialCache = new WeakMap();
+
```

Comment:
I think I would make these members of the loader, I think, since the materials are only relevant to the loader being used. Other than that I think this is good.